src/model/member.js

```diff
@@ -5,13 +5,26 @@ import responseBuilder from 'service/responseBuilder';
 import resource from 'model/resource';
 
 export default function member(name, id, parent) {
-    var refEndpoint = endpoint([parent.url(), name].join('/'), parent());
+    var url = parent.customUrl && parent.customUrl() ? parent.customUrl() : [parent.url(), name, id].join('/');
+
+    var refEndpoint = endpoint(url, parent());
 
     var model = {
+        _url: null,
+
+        customUrl(url) {
```

Comment:
Collection extend resource but don't have `one()`and `all()` method, so we can't factorize (cc @RobinBressan).